docs/volume-expansion-guide.md

```diff
@@ -0,0 +1,205 @@
+    <Instance IP>@tcp:/lfs51954  8.7T   11M  8.6T   1% /lustre_volume
+    ```
+
+7. Get the zone information from the `volumeHandle` of PV spec
+    ```console  
+    $ kubectl get pv <PV_NAME> -o yaml
+    ```
+    ```yaml
+    apiVersion: v1
+    kind: PersistentVolume
+    metadata:
+    annotations:
+      pv.kubernetes.io/provisioned-by: lustre.csi.storage.gke.io
+      volume.kubernetes.io/provisioner-deletion-secret-name: ""
+      volume.kubernetes.io/provisioner-deletion-secret-namespace: ""
+    creationTimestamp: "2025-10-27T23:02:27Z"
+    finalizers:
+    - external-provisioner.volume.kubernetes.io/finalizer
+    - kubernetes.io/pv-protection
+    name: <PV_NAME>
+    resourceVersion: "89077"
+    uid: 8e5b6022-210d-449c-a91a-9d1a0d49427e
+    spec:
+    accessModes:
+    - ReadWriteMany
+    capacity:
+      storage: 9000Gi
+    claimRef:
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      name: lustre-pvc
+      namespace: default
+      resourceVersion: "73594"
+      uid: 49777948-561d-4361-b7ee-162c6ca7be78
+    csi:
+      driver: lustre.csi.storage.gke.io
+      volumeAttributes:
+      filesystem: lfs51954
+      ip: <Instance IP>
+      storage.kubernetes.io/csiProvisionerIdentity: 1761597905856-8018-lustre.csi.storage.gke.io
+      volumeHandle: ${PROJECT_ID}/us-central1-c/<PV_NAME>
+    persistentVolumeReclaimPolicy: Delete
+    storageClassName: lustre-rwx
+    volumeMode: Filesystem
+    status:
+      lastPhaseTransitionTime: "2025-10-27T23:02:27Z"
+      phase: Bound
+    ```
+
+8. Verify the lustre instance properties
+
+    ```console
+    $ gcloud lustre instances describe <PV_NAME> --location=us-central1-c
+    ```
+
+    ```yaml
+    capacityGib: '9000'
+    createTime: '2025-10-27T22:38:13.236544672Z'
+    filesystem: lfs51954
+    labels:
+      kubernetes_io_created-for_pv_name: <PV_NAME>
+      kubernetes_io_created-for_pvc_name: lustre-pvc
+      kubernetes_io_created-for_pvc_namespace: default
+    storage_gke_io_created-by: lustre_csi_storage_gke_io
+    mountPoint: 10.97.160.4@tcp:/lfs51954
+    name: projects/${PROJECT_ID}/locations/us-central1-c/instances/<PV_NAME>
+    network: projects/${PROJECT_ID}/global/networks/${NETWORK_NAME}
+    perUnitStorageThroughput: '1000'
+    state: ACTIVE
+    uid: d2d5cf79-6bd0-41e9-a43f-a642bc37c9b3
+    updateTime: '2025-10-27T23:01:51.109896688Z'
+    ```
+
+9. Expand volume by modifying the field `spec -> resources -> requests -> storage` in PVC. Refer to the Managed Lustre performance tiers [documentation](https://cloud.google.com/managed-lustre/docs/create-instance#performance-tiers) to select a new storage size based on the throughput.
+    
+    ```
+    $ kubectl edit pvc lustre-pvc
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    ...
+    spec:
+      resources:
+        requests:
+          storage: 18000Gi
+      ...
+    ...
+    ```
+
+10. Confirm that the new storage size is reflected in PV and PVC
```

Comment:
Do we also want to mention the intermediate state before the expansion completes, as well as the estimated completion time?